test/fixtures/project/client/abc/empty-expected.js

```diff
@@ -0,0 +1,3 @@
s.call(this,arguments[1],arguments[2]);break;default:var t=arguments.length,n=new Array(t-1);for(var r=1;r<t;r++)n[r-1]=arguments[r];s.apply(this,n)}return!0}if(s){var t=arguments.length,n=new Array(t-1);for(var r=1;r<t;r++)n[r-1]=arguments[r];var a=s.slice();for(var r=0,t=a.length;r<t;r++)this.event=e,a[r].apply(this,n);return!0}},o.prototype.on=function(e,s){this._events||i.call(this),this.emit("newListener",e,s);if(this.wildcard)return a.call(this,e,s),this;if(!this._events[e])this._events[e]=s;else if(typeof this._events[e]=="function")this._events[e]=[this._events[e],s];else if(n(this._events[e])){this._events[e].push(s);if(!this._events[e].warned){var o;this._events.maxListeners!==t?o=this._events.maxListeners:o=r,o&&o>0&&this._events[e].length>o&&(this._events[e].warned=!0,console.error("(node) warning: possible EventEmitter memory leak detected. %d listeners added. Use emitter.setMaxListeners() to increase limit.",this._events[e].length),console.trace())}}return this},o.prototype.onAny=function(e){this._all||(this._all=[]);if(typeof e!="function")throw new Error("onAny only accepts instances of Function");return this._all.push(e),this},o.prototype.addListener=o.prototype.on,o.prototype.off=function(e,t){if(typeof t!="function")throw new Error("removeListener only takes instances of Function");var r;if(this.wildcard){var i=typeof e=="string"?e.split(this.delimiter):e.slice(),s=u.call(this,null,i,this.listenerTree,0);if("undefined"==typeof s)return this;r=s._listeners}else{if(!this._events[e])return this;r=this._events[e]}if(n(r)){var o=-1;for(var a=0,f=r.length;a<f;a++)if(r[a]===t||r[a].listener&&r[a].listener===t||r[a]._origin&&r[a]._origin===t){o=a;break}if(o<0)return this;this.wildcard?s._listeners.splice(o,1):this._events[e].splice(o,1),r.length===0&&(this.wildcard?delete s._listeners:delete this._events[e])}else if(r===t||r.listener&&r.listener===t||r._origin&&r._origin===t)this.wildcard?delete s._listeners:delete this._events[e];return this},o.prototype.offAny=function(e){var t=0,n=0,r;if(e&&this._all&&this._all.length>0){r=this._all;for(t=0,n=r.length;t<n;t++)if(e===r[t])return r.splice(t,1),this}else this._all=[];return this},o.prototype.removeListener=o.prototype.off,o.prototype.removeAllListeners=function(e){if(arguments.length===0)return!this._events||i.call(this),this;if(this.wildcard){var t=typeof e=="string"?e.split(this.delimiter):e.slice(),n=u.call(this,null,t,this.listenerTree,0);if("undefined"==typeof n)return this;n._listeners=null}else{if(!this._events[e])return this;this._events[e]=null}return this},o.prototype.listeners=function(e){if(this.wildcard){var t=[],r=typeof e=="string"?e.split(this.delimiter):e.slice();return u.call(this,t,r,this.listenerTree,0),t}return this._events||i.call(this),this._events[e]||(this._events[e]=[]),n(this._events[e])||(this._events[e]=[this._events[e]]),this._events[e]},o.prototype.listenersAny=function(){return this._all?this._all:[]},e.EventEmitter2=o}(typeof n=="undefined"?window:n)});
```

Comment:
Line is too long.<br>Missing '()' invoking a constructor.<br>Missing semicolon.<br>Too many errors. (66% scanned).